src/api.js

```diff
@@ -32,3 +32,16 @@ export const getArticle = (article_id) => {
       throw error;
     });
 };
+
+export const getComments = (article_id) => {
+  return NewsApi.get(`/articles/${article_id}/comments`)
+    .then(({ data: { comments } }) => {
+      console.log(comments);
+      console.log(typeof comments);
+      return comments;
+    })
+    .catch((error) => {
```

Comment:
refer to Stephen's comment in slack, catching here is not ideal as you are resolving the promise even if something goes wrong.